content/apps/hell.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Dynamic Links: Hell AP
+parent: Applications
+nav_order: 60
+---
+
+# Hell Attachment Point (Hell AP)
```

Comment:
I wonder whether a more concrete name would help users find this? Like "Degraded Connectivity Testbed" or similar.